Staking Page Reorg

DrW3RK opened the pull request:
Bringing in changes made in https://github.com/w3f/polkadot-wiki/pull/3623

DrW3RK committed:
Staking Page Reorg

DrW3RK committed:
Add risk rewards image

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+:::info
+
+For ledger users staking directly on Ledger Live, there is currently only one option to use one
+account as both stash and controller.
```
```suggestion
For ledger users staking directly on Ledger Live, currently there is no option to use separate stash and controller accounts.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+
+This hierarchy of separate key types for stash and controller accounts was designed so that nominators and validator operators can
+protect themselves much better than in systems with only one key. As a rule, the more often one uses
+a private key the higher its visibility and thus the chance it can be stolen. So, if one uses a key
```
```suggestion
This hierarchy of separate keys for stash and controller accounts was designed to add a layer of protection to nominators and validator operators. The more often one exposes and uses a private key, the higher its vulnerability for hacks or scams. So, if one uses a key
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+protect themselves much better than in systems with only one key. As a rule, the more often one uses
+a private key the higher its visibility and thus the chance it can be stolen. So, if one uses a key
+for multiple roles on a blockchain network, security can be easily compromised as the likelihood one
+uses that key often is high. Note that the damage linked to stolen private keys is different
```
```suggestion
for multiple roles on a blockchain network, it is likely that the account can get compromised. Note that the damage linked to stolen private keys is different
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+### Staking proxies
+
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is built using
+[substrate](https://substrate.io/), a modular system to efficiently build blockchains without having necessarily deep knowledge about blockchain technology. Within each module or `pallet`, one can `call` different
```
```suggestion
[substrate](https://substrate.io/), a modular system to efficiently build blockchains. Within each module or `pallet`, one can `call` different
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+functions that have similar logic. For example, the staking pallet contains all functionalities
+related to staking such as bonding or unbonding funds. The combined information of pallets and calls
+constitutes an `extrinsic`, i.e. a transaction that is executed from outside the chain but that
+triggers an event within the chain. Continuing with the staking example, within the staking pallet
```
```suggestion
triggers an event on the chain. Continuing with the staking example, within the staking pallet
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+related to staking such as bonding or unbonding funds. The combined information of pallets and calls
+constitutes an `extrinsic`, i.e. a transaction that is executed from outside the chain but that
+triggers an event within the chain. Continuing with the staking example, within the staking pallet
+one can bond funds and nominate some validators. The signature of such
```
Are you sure about bonding funds and nominating validators triggers rewards payouts? I think the idea you are trying to convey here will need rephrasing.

filippoweb3 replied:
Re-phrased

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+having special permissions also called **proxy accounts**.
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
+accounts to which one can delegate signatures to calls from specific pallets. There is thus the
```
```suggestion
In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
accounts which can sign extrinsic calls made to specific pallets on behalf of the proxied account. There is thus the
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
+accounts to which one can delegate signatures to calls from specific pallets. There is thus the
+possibility to create staking proxy accounts that can be used to sign only calls from staking,
+session and utility pallets. This makes the stash account even more isolated than using a controller
```
```suggestion
possibility to create staking proxy accounts that can be used to sign only extrinsic calls to staking,
session and utility pallets. This makes the stash account even more isolated than using a controller
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+accounts to which one can delegate signatures to calls from specific pallets. There is thus the
+possibility to create staking proxy accounts that can be used to sign only calls from staking,
+session and utility pallets. This makes the stash account even more isolated than using a controller
+account since now one can bond / unbond / bond more funds using the staking proxy account. However,
```
```suggestion
account since one can bond / unbond / bond more funds using the staking proxy account. However,
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+account since now one can bond / unbond / bond more funds using the staking proxy account. However,
+it is important to remember that actions on proxy accounts are limited, and in the case of staking
+proxy accounts, calls from the balance pallet cannot be signed. This means that it is not possible to
+send funds from a staking proxy. To do that one needs to remove that account as a staking proxy.
```
```suggestion
it is important to remember that actions that can be performed by the proxy accounts are limited, and in the case of staking
proxy, extrinsic calls to the balances pallet cannot be signed. This means it is not possible to do balance transfers
on the proxied account through a staking proxy.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
-objectives of this election mechanism are to maximize the security of the network, and achieve fair
-representation of the nominators. If you want to know more about how NPoS works (e.g. election,
-running time complexity, etc.), please read
+The solution to the election of nominators has to meet certain requirements, such as maximizing the
```
"The solution of the election of nominators" is ambiguous. The election solution comprises of both nominators and validators. It is primarily for electing validators for the active set

```suggestion
The "election solution" which is a connected graph between nominators and validators with the stake as edge weights, has to meet certain requirements, such as maximizing the
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+:::
 
-### 4. Rewards Mechanism
+Validator pools are paid the same regardless of stake level. Pools with less stake will generally pay more to nominators
```
```suggestion
Validators are paid the same regardless of stake backing them. Validators with less stake will generally pay more to nominators
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
 
-### 4. Rewards Mechanism
+Validator pools are paid the same regardless of stake level. Pools with less stake will generally pay more to nominators
+per-token than pools with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower-staked validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validator pools will be as evenly distributed as possible, to avoid a concentration of power among a few validators. In the long term, validator pools will have  ...
```
```suggestion
per-token than the ones with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower-staked validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validators will be as evenly distributed as possible which avoids concentration of power among a few validators. In the long term, validators will have similar levels of stake, with the stake
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+per-token than pools with more stake. This gives nominators an economic incentive to gradually shift their preferences to lower-staked validators that gain a sufficient amount of reputation. A consequence of this is that the stake across validator pools will be as evenly distributed as possible, to avoid a concentration of power among a few validators. In the long term, validator pools will have  ...
+being higher for higher reputation validators (meaning that a nominator that is willing to risk more
+by backing a validator with a low reputation will get paid more).
 
```
```suggestion
being higher for validators with higher reputation. A nominator who is willing to risk more
by backing a validator with a lower reputation will get paid more, provided there are no slashing events.

```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+Before distributing rewards to nominators, validators can create a cut of the reward (a commission) that is not shared with the nominators.
+This cut is a percentage of the block reward, not an absolute value. After the commission gets
+deducted, the remaining portion is based on their staked value and split between the validator and
+all of the nominators who have voted for this validator.
```
```suggestion
deducted, the remaining portion is distributed pro-rata based on their staked value and split between the validator and
all of the nominators whose stake has backed this validator.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
+`validator_commission = 50%`, in which case the validator would receive 5 DOT. The remaining 5 DOT
+would then be split between the validator and their nominators based on the proportion of stake each
+nominator had. Note that validators can put up their own stake, and for this calculation, their
+stake acts just as if they were another nominator.
```
```suggestion
nominator had. Note that for this calculation, validator's self-stake acts just as if they were another nominator.
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,364 @@ slug: ../learn-staking
-In the long term, we expect all validator pools to have similar levels of stake, with the stake
-being higher for higher reputation validators (meaning that a nominator that is willing to risk more
-by backing a validator with a low reputation will get paid more).
+Within a validator pool, a percentage of the reward goes thus to pay the validator's
```
```suggestion
Thus, a percentage of the reward goes thus to pay the validator's
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -221,89 +396,89 @@ The following example should clarify the above. For simplicity, we have the foll
+are essentially "wasting" their stake - they used their nomination to elect that validator to the
+active stake, but receive no rewards in exchange for doing so.
+
+Note that the network slashes a validator slot for a misbehavior (e.g. validator
```
```suggestion
Note that the network slashes a validator for a misbehavior (e.g. validator
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -221,89 +396,89 @@ The following example should clarify the above. For simplicity, we have the foll
 validators whom they consider to be trustworthy.
 
-The second point to note is that each validator candidate is free to name their desired commission
+Also note that each validator candidate is free to name their desired commission
```
```suggestion
Also, note that each validator candidate is free to name their desired commission
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -221,89 +396,89 @@ The following example should clarify the above. For simplicity, we have the foll
-  validator, it also sets your [session keys](learn-keys.md#session-keys). It only needs enough
-  funds to pay transaction fees.
+In order to be paid your staking rewards, someone must claim them for each validator that you
+nominate. Staking rewards are kept available for 84 eras, which is approximately {{ polkadot: 84 days :polkadot }}{{ kusama: 21 days :kusama }}. For more information on why this is so, see the page on [simple payouts](learn-simple-payouts.md).
```
> In order to be paid your staking rewards, someone must claim them for each validator that you
nominate.

A bit ambiguous

```suggestion
The distribution of staking rewards to the nominators is not automatic and needs to be triggered by someone. Typically the validators take care of this, but anyone can permissionlessly trigger rewards payout for all the nominators whose stake has backed a specific validator in the active set of that era. Staking rewards are kept available for 84 eras, which is approximately {{ polkadot: 84 days :polkadot }}{{ kusama: 21 days :kusama }}. For more information on why this is so, see the page on [simple payouts](learn-simple-payouts.md).
```

filippoweb3 replied:
I agree thanks for the suggestion

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -221,89 +396,89 @@ The following example should clarify the above. For simplicity, we have the foll
-Validators do most of the heavy lifting: they produce new block candidates in BABE, vote and come to
-consensus in GRANDPA, validate the state transition function of parachains, and possibly some other
-responsibilities regarding data availability and [XCM](learn-cross-consensus.md).
+:::warning Time limit
```
```suggestion
:::warning Time limit to claim staking rewards
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -221,89 +396,89 @@ The following example should clarify the above. For simplicity, we have the foll
-![staking](../assets/NPoS/staking-keys_stash_controller.png)
+Rewards are recorded per session (approximately {{ polkadot: 4 hours :polkadot }}{{ kusama: one hour :kusama }}) and
+calculated per era (approximately {{ polkadot: 24 hours :polkadot }}{{ kusama: 6 hours :kusama }}). Thus,
+rewards will be calculated {{ polkadot: once per day :polkadot }}{{ kusama: 4 times per day :kusama }}. Rewards are calculated based on era points, which have a probabilistic component. In other words,
```
This was from an earlier PR

```suggestion
Rewards are calculated per era (approximately six hours on Kusama and twenty-four hours on Polkadot). These rewards are calculated based on era points, which have a probabilistic component. In other words,
```

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -221,89 +396,89 @@ The following example should clarify the above. For simplicity, we have the foll
+:::
 
-### Want to stake DOT?
+Rewards can be directed to the same account used to sign the payout (controller), to the stash account (and either
```
```suggestion
Rewards can be directed to the same account used to sign the payout (controller), or to the stash account (and either
```

DrW3RK reviewed (commented):
Final feedback from my end for the Staking doc.

- Run grammarly after the suggestions are committed
- If possible, replace references to "validator pools" from the document with an appropriate wording

@emresurmeli Please read the whole document like a new user who is trying to learn about Staking on Polkadot and make suggestions accordingly

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
Re-phrased sentence as suggested by Radha

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

filippoweb3 committed:
Update docs/learn/learn-staking.md

Co-authored-by: Radha <86818441+DrW3RK@users.noreply.github.com>

Commit:
Merge branch 'staking-reorg' of https://github.com/w3f/polkadot-wiki into staking-reorg

Commit:
Rephrased paragraph for better clarity

DrW3RK commented:
@kianenigma Please take a final look at the PR that will replace majority of the content on the Staking page on the Wiki https://wiki.polkadot.network/docs/learn-staking
This follows the outline we narrowed upon during our last meeting https://github.com/w3f/polkadot-wiki/issues/3401

Commit:
Minor edits

- Run grammarly on the first section
- Replaced "check" with "see" when referring to support articles
- minor fixes

DrW3RK committed:
Improve SEO, styling and a few edits

DrW3RK committed:
Title edit

DrW3RK committed:
minor edits

Commit:
Added more explanation about era points

- added explanation about era points higher for para-validators, link to the maintain page on the wiki
- more grammar fixes

Commit:
Merge branch 'staking-reorg' of https://github.com/w3f/polkadot-wiki into staking-reorg

profradha commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,355 @@ slug: ../learn-staking
+compromise between the two, as minimizing risks might decrease rewards as well. Nominators should pay attention, especially to six criteria when nominating validators (not in order of importance):
+
+- recent history of the era points earned across eras
+- validator's (own) stake (shows skin in the game)
```
We refer to Validator's own stake as Self Stake https://wiki.polkadot.network/docs/maintain-guides-validator-community#self-stake You could also link to this doc here

filippoweb3 replied:
Alright then I need to change the figure. I took the terminology from the staking tab on the UI. I wanted anyway to add the figure for the staking proxy and the updated figure for the stash-controller.

profradha replied:
Got it. In that case, let us leave it as is. Should we may be add a snapshot of Polkadot JS UI along with the figure and pointing to the respective parameters?

filippoweb3 replied:
New Figures added, updated Figure risk-rewards 👍

Commit:
New Figures

- updated figure risk-rewards
- new figure stash-controller
- new figure stash - staking proxy
- minor edits

Commit:
Added old part on slashing

Added part that is not present elsewhere on the wiki. Initially deleted, I noticed that the information is only partly present on other wiki articles.

Commit:
Minor edits

- finished to run grammarly
- fetched number of active validators

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
 
```
Do we have resources to have somewhat better grapghics? If not, can we at least use the Polkadot font? a proper background color that's Polkadot themed?

filippoweb3 replied:
Hi @kianenigma I am using google docs to create the graphics and it does not allow to install the Polkadot font. I will need to see what are other options with W3F. I initially produced the images with Polkadot branding colour as background but I switched to a white background using the branding colour for the font. It seemed "cleaner" to me. But I can revert it. What do you mean exactly with "better graphics"? Is the overall presentation of the diagrams not good, or just the font and the background colour?

profradha replied:
Let me check with the design team on this. We were already thinking about revamping the graphics on the Wiki

filippoweb3 replied:
I spoke to Iggy and he told me to use Inter font for the figures. Polkadot unbounded font is only used for titles. I updated all figures accordingly.

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
 {{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
-validators as trusted validator candidates. Both validators and nominators lock their tokens as
-collateral and receive staking rewards.
+validators as trusted validator candidates, and the network will automatically distribute the stake among validators in
```
autoamtically, such that the economic security is maximized. 

And ideally "economic security" should be explained in:

```
Similar to the miners in PoW networks, PoS networks have...
```

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+blocks on the chain. Consensus is the process of agreeing on something, in this case, the progression
+of the blockchain or how blocks are added to the chain. Consensus consists of two actions:
+
+- `block production`, i.e. the way multiple blocks candidates are produced, and 
```
I would not use code formatting for what is not code.

filippoweb3 replied:
Fixed throughout the page

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+- `block production`, i.e. the way multiple blocks candidates are produced, and 
+- `block finality`, i.e. the way only one block out of many candidates is selected and added to the canonical chain (see [this](learn-consensus.md/#probabilistic-vs-provable-finality) article for more information about finality).
+
+Proof-of-Work (PoW) and Proof-of-Stake (PoS) are well-known mechanisms used to reach consensus in a secure and trustless way on public blockchains, where there are many participants who do not know each other (and probably never will). In PoW, network security relies on the fact that the miners who are responsible for adding blocks to the chain must compete to solve difficult mathematic puzzles t ...
```
one paragraph for each would be a cleaner description, and I generally think both can be a bit more concise (but that subjective).

filippoweb3 replied:
Suggestion accpeted

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+### Nominating Validators
+
+The action of nominating consists of a) locking or
+bonding tokens (stake) on the chain, and b) nominating a set of validator candidates to whom the stake will
```
make a proper markdown list please?

filippoweb3 replied:
Done

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+production during a specific `era`. An era is a period of {{ polkadot: 24 hours :polkadot }}{{ kusama: 6 hours :kusama }} during
+which an `active set` of validators is producing blocks and performing other actions on chain. This
+means that not all validators are in the active set and such set changes between eras. Each era is
+divided into 6 epochs or `sessions` during which validators are assigned as block producers to
```
in the spirit of conciseness, maybe we can skip epoch and session details here. 

Who is the audience here? What I am reading is not super advance, but it is also not geared towards a non-crypto-native end user. Is that a fair assumption?

filippoweb3 replied:
@kianenigma I introduced the terms here, and defined them appropriately because they are used later in the page and I believe is important for the reader to know what these things are in general, and they must know it as soon as possible before the explanations become complex. Let me know if you agree with this.

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+
+### Tasks and Responsibilities of a Nominator
+
+Since validator slots are limited, most of those who wish to stake their DOT and contribute to the
```
DOT should be DOT / KSM?

filippoweb3 replied:
yep, thanks. This is fixed now.

profradha replied:
We can also conditionally render this doc for Polkadot and Kusama

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+commission at any time), and general health monitoring of their validators' accounts. Thus, while not
+being completely set-it-and-forget-it, a nominator's experience is relatively hands-off compared to that of
+a validator. For more information, you can take a look at the [nominator guide](learn-nominator.md)
+to understand your responsibilities as a nominator.
```
Here you should mention that one step more passive than nomination is joining a pool.

filippoweb3 replied:
Added

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+
+If you want to become a nominator, see
+[this](../maintain/maintain-guides-how-to-nominate-polkadot.md) guide. If you are a beginner and
+would like to securely stake your tokens using Polkadot JS Apps, refer to [this](https://support.polkadot.network/support/solutions/articles/65000168057-how-do-i-stake-nominate-on-polkadot-) support article. {{ kusama: The tutorial presented in the support article is demonstrated on Polkadot, but the procedure is the same for Kusama :kusama }}
```
do we have an equivalent in the staking dashboard?

profradha replied:
@filippoweb3 Please link to this article as well https://support.polkadot.network/support/solutions/articles/65000182133-how-to-use-the-staking-dashboard-staking-your-dot

filippoweb3 replied:
Added

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+
+### Selection of Validators
+
+The task of choosing validators is not simple, as it should take into account nominator reward and
```
have we considered justifying the rendered text? it probably makes it a lot more readable.

kianenigma commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
 
 :::
 
+### Bags List
```
We should move this as low/far as possible, because ideally users don't have to deal with this. Can it be moved further down, or to a separate page?

filippoweb3 replied:
I agree. Moved at the end of the Section "Staking System Overview"

kianenigma reviewed (changes requested):
Much better, but I think we can do even better. 

First, the overall writing is good, but I do have a subjective comment that it is not super concise. As if the goal is to put as many keywords in a sentence as we can, while I think it should be the opposite. I think we should keep the article as short, simple and concise as we can.

I know this is subjective, but it is the undeniable feedback that we know is out there: Polkadot is too complicated, and people are having a hard time understanding it. I don't think we should "dumb it down" (say wrong things for the sake of simplicity), but we can work on having a simpler wiki.

As for the reading, I read more than half so far, and aside form the comments, I think the TOC makes a lot of sense until section 2:

<img width="352" alt="Screen Shot 2022-09-14 at 19 42 37" src="https://user-images.githubusercontent.com/5588131/190247981-05712077-4138-4804-bf6e-23989480bbd0.png">

"Being a Nominator"

After that, the axonomy seems very random to me. Why is chilling a standalone top-level section, and not a subset of "being a nominator"? 

What is exactly "Staking system overview"? anything that didn't fit in the other categories? 

I think you should begin by explaining the 3 roles that one can have in staking: 

1. chilled 
2. validator
3. nominator

and the two main events that can happen: 

1. reward 
2. slash

and come up with a ToC order based on that.

(above is just a suggestion).

kianenigma commented:
I think what I said above is particularly important because the above page is the ONLY staking page in the wiki as far as I can see. this is really tricky, because we will have to mention everything, including something super advance like bags-list next to a rudimentary thing like "what is staking rewards".

We should really break this down into 

1. Intro to staking 
2. Advance Staking
3. Nomination Pools

Commit:
Addressed part of Kian's feedback

- mentioned economic security in the intro
- removed code formatting for non-coding stuff
- removed term "validator pools" as it confuses nominator pools
- Divided PoW and PoS into two paragraphs in the Intro
- markdown list for nominating steps
- added DOT/KSM shift where missing
- Added info about joining nomination pools
- Moved bags-list section further down

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+
+### Nominating Validators
+
+The action of nominating consists of a) locking or
```
The "lock", "bond" and "stake" terms tend to be used interchangeably. Maybe simplify to:

Nominating on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} requires 2 actions:

 a) Locking tokens on-chain.
 b) Selecting a set of validators, to whom these locked tokens will automatically be allocated to. 

How many tokens you lock up is completely up to you - as are the validators you wish to select.

_Locking tokens is also known as **bonding**.  You can also refer to your locked tokens as your **bonded** tokens, or **staked** tokens. Likewise, selecting validators is also known as **backing** or **nominating** validators. These terms are used interchangeably by the community._

From now on locked tokens will be referred to as **bonded** tokens.

Once the previous 2 steps are completed and you are nominating, your bonded tokens could be allocated to one or more of your selected validators, and this happens _every time the active **validator set** changes_. 

This validator set is updated every **era** on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}. On Polkadot, an era is 1 day.

Unlike other staking systems, Polkadot automatically chooses which of your selected validators will be backed by your bonded tokens. Selecting a group of validators increases your chances of consistently backing at least one who is active. This results in your bonded tokens being allocated to validators more often, which means more network security and more rewards.

This is in stark contrast to other staking systems that only allow you to back one validator; if that validator is not active, you as a staker will also not be. 

Polkadot's nomination model solves this.

Commit:
Added new page "Advanced staking"

- added "Advanced staking" page
- modified sidebars files for Polkadot and Kusama
- started modifying staking titles

Commit:
Moved staking proxies to "Advanced staking"

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+staking process can be punished or `slashed`, and depending on the gravity of the situation, their
+stake can be partly or fully confiscated by the network. It is not in a staker's economic interest to orchestrate an attack and risk losing tokens. Any rational actor staking on the network would want to get rewarded, and the PoS network rewards good behavior and punishes bad behavior.
+
+## Nominated Proof-of-Stake Overview
```
Is it worth having a section here to distinguish the difference between nominators, validators and pools? Perhaps it would be good to say Pools is "built" on top of NPoS to provide a very low barrier to entry to staking, without sacrificing Polkadot's strict security model.

DrW3RK commented:
@kianenigma Thanks for the detailed feedback. We will work on it

rossbulat reviewed (commented):
Noted a couple of things to clarify nearer the top of the article. Most notably a more streamlined introduction to what nominating is.

Commit:
Moved bags-list to advanced staking

Commit:
Removed "Staking system overview" section

Removed the Section and moved the content at the beginning of the page when explaining NPoS

Commit:
Section "Rewards distribution" moved to advanced staking

Theory behind rewards distribution is not relevant to average stakers, who will use nomination pools.

Commit:
Moved detailed info about slashing to advanced staking page

Info about unresponsiveness, equivocation and slashing across eras is not relevant for average stakers.

Commit:
Chilling added to "Being a nominator"

Commit:
Inflation info moved to advanced staking

Info about inflation does not affect average nominators choice in staking

Commit:
Why staking modified

- The section moved up in title rank
- "Why" and "why not" merged in one section

Commit:
Added staking dashboard intormation

Commit:
Added Ross suggestion

Added section about locking/bonding

Commit:
Minor fixes

Checked throughout the doc that "bonded' is used

Commit:
Validators, nominators, pools

As requested by Ross:

- Divided the sections between the three
- Added "Pools"

filippoweb3 commented:
@kianenigma @rossbulat @DrW3RK  all feedbacks implemented. Thanks for the detailed reviews! I am now checking all internal links with titles / Sections and I will have a last read through it.

Commit:
Added figures with Polkadot font

Changed font from ARIAL to INTER, as discussed with Iggy

Commit:
Fixed links

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -1,324 +1,390 @@
 {{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}
-is required to submit an intention to nominate. The nomination intents are placed in a semi-sorted
-list called [bags-list](https://github.com/paritytech/substrate/pull/9507).
+is required to submit an intention to nominate. Note that in NPoS the stake of both nominators and
```
...is required to submit an intention to nominate, **which can be thought of as registering to be a nominator.**

(trying to make clear what "intention" is.

filippoweb3 replied:
Specified

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -1,324 +1,390 @@
 which is a dynamic value that can be much higher than
-{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>. :polkadot }}
-{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/>. :kusama }}
+{{ polkadot: <RPC network="polkadot" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="query.staking.minNominatorBond" defaultValue={100000000000} filter="humanReadable"/> :kusama }}.
```
This dynamic value depends on the amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} being staked, in addition to the selected nominations.

filippoweb3 replied:
Added

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -1,324 +1,390 @@
+monitoring their performance, keeping an eye on changing commission rates (a validator can change
+commission at any time), and general health monitoring of their validators' accounts. Thus, while
+not being completely set-it-and-forget-it, a nominator's experience is relatively hands-off compared
+to that of a validator, and even more with [nominator pools](./learn-nomination-pools.md). For more
```
change nominator pools to nomination pools.

filippoweb3 replied:
Done

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -1,324 +1,390 @@
+
+For Ledger users staking directly on Ledger Live, currently, there is no option to use separate
+stash and controller accounts. That is if you stake on Ledger Live your stash account will be your
+controller too.
```
Yes this is an issue. We might want to add that Ledger is now supported in Talisman extension. Users can import their Ledger accounts in there and use them as a stash and controller.

Ledger still doesn't support key extrinsics like identity though (2022, shocking!) which is why I personally hold off from using Ledger.

filippoweb3 replied:
Added

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -327,154 +393,44 @@ and slashes are not mitigated by having other inactive or waiting nominations. T
-actively validating. Any nominator actively nominating BC_2 also receives a 1% slash, but any
-nominator actively nominating OV_1 is unaffected.
-
 In rare instances, a nominator may be actively nominating several validators in a single era. In
```
Or with very large bonds such as parachain liquid staking accounts (Acala's LDOT nominator typically has 7-12 active nominations per era).

filippoweb3 replied:
Interesting. Added in the text.

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -489,170 +445,11 @@ Polkadot allows some validators to be disabled, but if the number of disabled va
-For those who are interested in knowing more about the design of inflation model for the network,
-please see
-[here](https://w3f-research.readthedocs.io/en/latest/polkadot/overview/2-token-economics.html).
+## Why and why not staking?
```
Is "Why and why not to stake?" more natural?

filippoweb3 replied:
Changed

rossbulat commented on docs/learn/learn-staking.md:
```diff
@@ -680,8 +477,7 @@ Polkadot started with 20 open validator positions and has increased gradually to
 the number of validators has not been determined yet, but should only be limited by the bandwidth
```
Is this outdated?

filippoweb3 replied:
probably. Removed.

rossbulat reviewed (changes requested):
Just a few little things, overall very high quality. Happy to approve post this round.

Commit:
Implemented feedback

Implemented part of Ross feedback

kianenigma reviewed (commented):
Don't we have a new recent Font for Polkadot? I think the one currently used are the old one. 

And, I am not a designer, but I really think we can do better here, even if we don't use external resources here. That white background with all ping text is just a bit too harsh on the eye.

Commit:
Added info talisman

as suggested by Ross

filippoweb3 commented:
@kianenigma let me propose 1-2 variations. It is good that we this here for this PR, so that we can use it as model to correct other pages / tutorial slides.

The Polkadot font is used for titles only. I was told I must use INTER font for text. I am creating diagrams in google docs with no possibility to import the UNBOUNDED font. If you have any suggestions of free software I can use and import the UNBOUNDED font and create diagrams, I would be happy to try.

filippoweb3 commented:
@kianenigma Is this version better? I took the background color from the Polkadot website.

![Screenshot 2022-09-19 at 13 45 30](https://user-images.githubusercontent.com/110459737/191010172-3bcc3ada-4837-43b4-994d-d784d6fc79d7.png)

filippoweb3 commented:
Here, another version. I can play a little with text size and the two colours. 

![Screenshot 2022-09-19 at 13 54 16](https://user-images.githubusercontent.com/110459737/191011536-b08b4e85-154f-466a-bdca-73004287a5c6.png)

filippoweb3 commented:
Here another version with an additional colour I found on the Polkadot website. I can use those 3 colours (in addition to the background) to update all the current diagrams.

![Screenshot 2022-09-19 at 14 02 21](https://user-images.githubusercontent.com/110459737/191012872-245a18ae-0bc5-4825-beaa-93ebde910151.png)

Commit:
Prettified

filippoweb3 commented:
@DrW3RK @kianenigma @rossbulat can you please have a look at the last PR edits? I suggested some alternatives (messages above) for the figures. I noticed there was a new issue opened about mentioning validator pools. This has been previously fixed in this PR. There are a lot of changes here and the staking page after this PR will be very different from the one online now.

rossbulat commented on docs/learn/learn-staking-advanced.md:
```diff
@@ -0,0 +1,385 @@
+a nominator can bond funds and nominate some validators. The signature of such
+extrinsic might lead to an event on the chain such as a reward payout to that nominator at the end of an era; this is an event inside the chain. This way of having
+transactions categorized within pallets and functionalities makes it possible to create accounts
+having special permissions also called **proxy accounts**.
```
This explanation of Substrate pallets feels out of context with this section. Perhaps have a link here to a dedicated section on this, and just introduce proxy accounts as a utilisation of Substrate pallets.

filippoweb3 replied:
Created a dedicated section at the beginning of the page, with links to substrate.

rossbulat commented on docs/learn/learn-staking-advanced.md:
```diff
@@ -0,0 +1,385 @@
+
+![staking](../assets/staking/stash-stakingProxy.png)
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
```
With the above comment in mind, this feels like a natural starting point for the section.

filippoweb3 replied:
Made links to dedicated section at the start of the page

rossbulat commented on docs/learn/learn-staking-advanced.md:
```diff
@@ -0,0 +1,385 @@
+![staking](../assets/staking/stash-stakingProxy.png)
+
+In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} proxy accounts are special
+accounts which can sign extrinsic calls made to specific pallets on behalf of the proxied account. There is thus the
```
"extrinsic calls" and "specific pallets" would be good places to link to the explanations.

filippoweb3 replied:
yep, made link to the start of the page. Also added link to the wiki proxy page (also being updated on a separate PR)

rossbulat commented on docs/learn/learn-staking-advanced.md:
```diff
@@ -0,0 +1,385 @@
+account since one can bond / unbond / bond more funds using the staking proxy account. However,
+it is important to remember that actions that can be performed by the proxy accounts are limited, and in the case of staking
+proxy, extrinsic calls to the balances pallet cannot be signed. This means it is not possible to do balance transfers
+on the proxied account through a staking proxy.
```
Is there an additional resource to link to that documents all the limitations / functions of the various types of proxy?

filippoweb3 replied:
Added link to proxy page

rossbulat commented on docs/learn/learn-staking-advanced.md:
```diff
@@ -0,0 +1,385 @@
+If one receives staking rewards and the amount of staked tokens within the stash account increases
+over time, the position within a bag changes and may also result in a change of bag. This may also
+happen if accounts within the bag bond more tokens or unbond tokens, one's account position and the
+position of other accounts in the bags list might change. These changes are not done automatically.
```
, requiring the nominator to submit `rebag` extrinsics to update their position.

filippoweb3 replied:
Added

rossbulat commented on docs/learn/learn-staking-advanced.md:
```diff
@@ -0,0 +1,385 @@
+happen if accounts within the bag bond more tokens or unbond tokens, one's account position and the
+position of other accounts in the bags list might change. These changes are not done automatically.
+The `voterList` pallet comes with the extrinsic `putInFrontOf` which helps the node to move up in the
+bag. Also, the pallet comes with an important permissionless extrinsic: `rebag`. This
```
aforementioned extrinsic: `rebag`

rossbulat commented:
@filippoweb3 I picked up on some small edits, but overall great detail! If there are any particular sections you specifically need eyes on please let me know & very happy to provide more feedback.

filippoweb3 commented:
@kianenigma what do you think about the versions I proposed for the diagrams?

filippoweb3 commented:
Thanks @rossbulat for the feedback. Can we merge this PR? I will open a new issue / and related PR to do the final minor edits. I feel like this PR is becoming big.

Commit:
Implemented Ross feedback

rossbulat commented:
> Thanks @rossbulat for the feedback. Can we merge this PR? I will open a new issue / and related PR to do the final minor edits. I feel like this PR is becoming big.

No probs! Yes that's a good idea, will be much easier to pick up on smaller edits.

filippoweb3 commented:
It seems we need another approval.

DrW3RK committed:
Merge branch 'master' into staking-reorg

kianenigma commented:
The merged graphics are the white ones, fwiw. 

I think this goes beyond this PR. I will make a separate issue.  https://github.com/w3f/polkadot-wiki/issues/3853
